escalate.gemspec

```diff
@@ -27,9 +27,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
-
-  # For more information and examples about making a new gem, checkout our
-  # guide at: https://bundler.io/guides/creating_gem.html
+  spec.add_dependency "activesupport"
+  spec.add_dependency "contextual_logger"
```

Comment:
I want to stay as lightweight as possible. Are you sure we need to depend on `contextual_logger`?

Reply:
We don't need to. This conversation is similar to the one below, so I'll resolve this one and we can discuss the road forward there.